maintenance_page/html/index.html

```diff
@@ -0,0 +1,103 @@
+              </svg>
+              <span class="govuk-header__logotype-text"></span>
+            </span>
+          </a>
+        </div>
+        <div class="govuk-header__content">
+          <a class="govuk-header__link govuk-header__service-name" href="/">Apply for teacher training</a>
+        </div>
+      </div>
+    </header>
+    <div class="govuk-width-container">
+      <main class="govuk-main-wrapper" id="main-content" role="main">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">This service is currently unavailable</h1>
+            <p class="govuk-body">We're sorry the service is not available right now.</p>
+            <p class="govuk-body">Try again later.</p>
+            <p class="govuk-body">If you reached this page after submitting information then it has not been saved. You will need to enter it again when the service is available.</p>
+            <p class="govuk-body">If you have any questions, email us at<br>
+              <a class="govuk-link" href="mailto:becomingateacher@digital.education.gov.uk">becomingateacher@digital.education.gov.uk</a>.
+            </p>
+          </div>
+        </div>
+      </main>
+    </div>
+    <footer role="contentinfo" class="govuk-footer govuk-!-display-none-print">
+      <div class="govuk-width-container">
+        <div class="govuk-footer__meta">
+          <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+            <h2 class="govuk-heading-m">Get help</h2>
+            <p class="govuk-body-s govuk-!-margin-bottom-1">Call 0800 389 2500 or <a class="govuk-link govuk-footer__link" href="https://getintoteaching.education.gov.uk/help-and-support?utm_source=apply-for-teacher-training.service.gov.uk&amp;utm_medium=referral&amp;utm_campaign=candidate_interface%2Fstart_page-create_account_or_sign_in&amp;utm_content=candidate_not_logged_in">chat online</a>.</p>
+            <p class="govuk-body-s govuk-!-margin-bottom-1">Monday to Friday, 8:30am to 5:30pm (except public&nbsp;holidays)</p>
+            <div class="govuk-!-margin-bottom-5">
+              <a class="govuk-footer__link" href="/candidate/make-a-complaint">Make a complaint or give feedback</a>
```

Comment:
@inulty-dfe I'm afraid these links will not work if the app is down. Unless we add them to the html, but this may be overkill.